src/DVRescueBundle/Resources/views/Default/license.html.twig

```diff
@@ -0,0 +1,56 @@
+{% extends 'DVRescueBundle::base.html.twig' %}
```

Comment:
I would like to see the license change settled and represented, as per here: https://github.com/mipops/dvrescue/issues/10, but that could be a subsequent PR.

Reply:
I show the current license in this PR so I suggest that I merge with it and change in another PR as soon as another license is chosen.